### PR TITLE
feat: Update homepage donation page background image and add footer

### PR DESCRIPTION
**Pull request by Manav2304**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -10,7 +10,9 @@
     "@types/node": "^16.18.13",
     "@types/react": "^18.0.28",
     "@types/react-dom": "^18.0.11",
+    "axios": "^1.3.4",
     "bootstrap": "^5.2.3",
+    "mdb-react-ui-kit": "^5.1.0",
```
Comment (vaidehi27) [inline package.json:15]: This library is used for what?

File: public/index.html [hunk 2]
```diff
@@ -2,14 +2,14 @@
 <html lang="en">
   <head>
     <meta charset="utf-8" />
-    <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
+    <link rel="icon" href="%PUBLIC_URL%/iskcon.png" />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
     <meta name="theme-color" content="#000000" />
     <meta
       name="description"
       content="Web site created using create-react-app"
```
Comment (vaidehi27) [inline public/index.html:10]: change the description as well

File: public/index.html [hunk 3]
```diff
@@ -31,6 +31,12 @@
       integrity="sha384-GLhlTQ8iRABdZLl6O3oVMWSktQOp6b7In1Zl3/Jr59b6EGGoI1aFkw7cmDA6j6gD"
       crossorigin="anonymous"
     />
+    <link
```
Comment (vaidehi27) [inline public/index.html:34]: Change the title as well at line 27

File: public/index.html [hunk 4]
```diff
@@ -31,6 +31,12 @@
...
+      href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css"
```
Comment (vaidehi27) [inline public/index.html:35]: why do we have 2 versions of bootstrap? `bootstrap@5.0.2` and `bootstrap@5.3.0-alpha1`?

File: src/styles.css [hunk 5]
```diff
@@ -0,0 +1,5 @@
+body {
+  background-image: url("C:\Users\lenovo\OneDrive\Desktop\New folder\ISKCON\src\images\image.png");
```
Comment (vaidehi27) [inline src/styles.css:2]: absolute path?

File: public/index.html [hunk 6]
```diff
@@ -2,14 +2,14 @@
 <html lang="en">
   <head>
     <meta charset="utf-8" />
-    <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
+    <link rel="icon" href="%PUBLIC_URL%/iskcon.png" />
```
Comment (vaidehi27) [inline public/index.html:5]: I don't see the iskon.png file. Where is it coming from?

File: src/App.tsx [hunk 7]
```diff
@@ -6,12 +6,13 @@ import {
   Link,
   BrowserRouter,
 } from "react-router-dom";
+import "./styles.css";
```
Comment (vaidehi27) [inline src/App.tsx:9]: css file? why?

File: src/Components/Footer/constant.ts [hunk 8]
```diff
@@ -0,0 +1,23 @@
+export const t1 = `Samadhi Arati – 04:10AM`;
```
Comment (vaidehi27) [inline src/Components/Footer/constant.ts:1]: what is t1, t2, t3? names are not self-explanatory. Give meaningful name

File: src/Components/Footer/index.tsx [hunk 9]
```diff
@@ -0,0 +1,143 @@
...
+          </a>
+          <a href="#" className="me-4 text-reset">
+            <MDBIcon color="secondary" fab icon="instagram" />
+          </a>
+          <a href="#" className="me-4 text-reset">
+            <MDBIcon color="secondary" fab icon="linkedin" />
+          </a>
+          <a href="https://t.me/iskconvvn" className="me-4 text-reset">
+            <MDBIcon color="secondary" fab icon="telegram" />
+          </a>
```
Comment (vaidehi27) [inline src/Components/Footer/index.tsx:39]: Can we use the map function here?

File: src/Components/Navbar/style.ts [hunk 10]
```diff
@@ -1,12 +1,38 @@
...
-  .nav-link {
-    color: dark black;
-    font-weight: 500;
-    margin-right: 1rem;
+  padding: 1.6rem;
+  display: flex;
+  justify-content: space-between;
+  // align-items: center;
+  // background-color: #fff;
+  // box-shadow: 0px 0px 10px rgba(0, 0, 0, 0.1);
```
Comment (vaidehi27) [inline src/Components/Navbar/style.ts:10]: remove commented lines

File: src/Components/Navbar/index.tsx [hunk 11]
```diff
@@ -4,25 +4,31 @@ import { NavStyle } from "./style";
...
-            <Link className="nav-link" to="#">
-              GO SEVA
-            </Link>
-          </div>
-        </nav>
-      </div>
+          <Link className="nav-link" to="#">
+            GO SEVA
+          </Link>
+          <Link className="nav-logo" to="/Pages/Home">
```
Comment (vaidehi27) [inline src/Components/Navbar/index.tsx:16]: why are we using class names? and not styled-components?

File: src/Pages/Donation/index.tsx [hunk 12]
```diff
@@ -3,17 +3,14 @@ import { DonationForm } from "./style";
...
-import { SubmitButton } from "./style";
 import { PageBackground } from "./style";
+import axios from "axios";
 
-interface DonationFormData {
-  name: string;
-  email: string;
-  amount: number;
-}
+const APP_ID = "33221729445f634e544edbc48c712233";
```
Comment (vaidehi27) [inline src/Pages/Donation/index.tsx:9]: app id should not be stored as a constant here. It should be used from .ev files. It's a secret that should not be pushed to git repo

File: src/Pages/Donation/index.tsx [hunk 13]
```diff
@@ -3,17 +3,14 @@ import { DonationForm } from "./style";
...
+const ORDER_ID = "<YOUR_ORDER_ID>";
```
Comment (vaidehi27) [inline src/Pages/Donation/index.tsx:10]: what is the order id here? Shouldn’t it be a number? BE doesn't give an error?

File: src/Pages/Donation/index.tsx [hunk 14]
```diff
@@ -24,18 +21,44 @@ const DonationPage: React.FC = () => {
...
   };
 
+  const handleDonateButtonClick = () => {
+    // Generate the payment form URL using the Cashfree API
+    const paymentFormUrl =
+      "https://payments-test.cashfree.com/forms/Iskcondonation" +
+      `?appId=${APP_ID}&orderId=${ORDER_ID}&orderAmount=${formData.amount}&orderCurrency=INR&customerName=${formData.name}&customerEmail=${formData.email}&customerPhone=9999999999`;
+
+    // Redirect the user to the payment form URL
+    window.location.href = paymentFormUrl;
```
Comment (vaidehi27) [inline src/Pages/Donation/index.tsx:31]: this is how you navigate to another page? Use navigate hooks from the react-router library.

File: src/Pages/Home/index.tsx [hunk 15]
```diff
@@ -1,108 +1,198 @@
 import React from "react";
 import { StyledCarousel } from "./style";
-import { Background } from "./style";
-
+import { aboutUs1, aboutUs2, aboutUs3, aboutUs4 } from "./constant";
+import { AboutUsText } from "./style";
+import {
+  Goals1,
```
Comment (vaidehi27) [inline src/Pages/Home/index.tsx:6]: What goals are we talking about here? Variable name is not self-explanatory

File: src/Pages/Home/index.tsx [hunk 16]
```diff
@@ -1,108 +1,198 @@
...
+  Goals3,
+  Goals4,
+  Goals5,
+  Goals6,
+  Goals7,
+  Goals8,
+} from "./constant";
+import {
+  History1,
+  History2,
```
Comment (vaidehi27) [inline src/Pages/Home/index.tsx:17]: What kind of history? Variable name is not self-explanatory

**Review by vaidehi27 — CHANGES_REQUESTED** [review]
1. Use the map function wherever necessary. 
2. Use styled components instead of the class names. 
3. Give proper names to variable
4. Check out react-hook-forms for form validation and how to handle form changes. You can also check out the project I share to see how I use the react-hook-form.
5. Remove commented code
6. Handle API errors properly. Check out the best practices on google

Overall very long and poor PR. Does a lot of things. The title of the PR is also not good. No punctuations

File: package.json (hunk 1 above)
Comment (Manav2304) [reply package.json:15]: This library is used for footer.

File: public/index.html (hunk 3 above)
Comment (Manav2304) [reply public/index.html:34]: ok

File: public/index.html (hunk 4 above)
Comment (Manav2304) [reply public/index.html:35]: I have removed unwanted version of Bootstrap

File: src/styles.css (hunk 5 above)
Comment (Manav2304) [reply src/styles.css:2]: it has been updated

File: src/App.tsx (hunk 7 above)
Comment (Manav2304) [reply src/App.tsx:9]: It is used to apply bg image for all the pages

File: src/Components/Footer/index.tsx (hunk 9 above)
Comment (Manav2304) [reply src/Components/Footer/index.tsx:39]: yes, I have updated it using map function

**Pull request closed without merging**